docs/anvil_setup/index.md

```diff
@@ -199,6 +199,12 @@ python3 -m pip install gpiod==1.5.4  # Install additional dependency that rosdep
 ./build.sh
 ./setup-drs.sh
 ```
+> [!Note]
+> `setup-drs.sh` will show three dialogs to be filled:
+> - ECU ID: the local ECU ID ( `0` or `1` ). Front and right facing sensor are connected to the ECU0, others are connected to the ECU1.
```

Comment:
Elsewhere in the document, they are referred to as ECU1 and ECU2. I guess 0 and 1 are correct?

Reply:
@drwnz 
Thank you very much for your detailed review. I checked the index manner through the document again and, as you suggested, found `0` and `1` appear more often.  I aligned the index manner by 1f1b5a25ec13faa020c3dc3ef1a6ed9aea0d0ee4 (replace 1/2 indexing with  0/1).
Thank you!